worker/charts/worker/templates/_helpers.tpl

```diff
@@ -126,4 +126,7 @@ NEOSYNC_CLOUD: {{ .Values.neosyncCloud.enabled | default "false" | quote }}
 {{- if and .Values.ee .Values.ee.license }}
 EE_LICENSE: {{ .Values.ee.license | quote }}
 {{- end }}
+{{- if .Values.tableSync.maxConcurrency }}
+TABLESYNC_MAX_CONCURRENCY: {{ .Values.tableSync.maxConcurrency }}
+{{- end }}
```

Comment:
The if wrapper is not required because you've set it as a default value in the values.yaml